nav2_costmap_2d/README.md

```diff
@@ -88,6 +89,12 @@ local_costmap:
 ```
 In order to add multiple sources to the global costmap, follow the same procedure shown in the example above, but now adding the sources and their specific params under the `global_costmap` scope.
 
+## Costmap Filters
+
+### Overview
+
+Costmap Filters - is a costmap layer-based instrument which provides an ability to apply to map spatial-dependent raster features marked at map-masks. These features are used in plugin algorithms when filling costmaps in order to allow robots to change their trajectory, behavior or speed when them (robots) enters/leaves an area marked in a map masks. More information about design, architecture of the feature and how it works could be found on navigation2 web-site: https://navigation.ros.org.
```

Comment:
"... marked **as** map-masks." 

"... when **a robot** enters/leaves ..."

"They can be used to create mask keepout zones or change maximum robot speeds based on the area it is near in a map."

Reply:
Just a one-liner on some applications of this that aren't as abstract as examples

Reply:
Fixed and added in https://github.com/ros-planning/navigation2/pull/1882/commits/ab3f5fe2e5f47906832bfee91c2b6ff8e280eb8d#diff-8959497cd0f5a2000ecd88ba923d1f87R96